STEP/CodeGeneration/CodeGenerationVisitor.cs

```diff
@@ -32,27 +32,46 @@ private void EmitAppend(string line)
         _stringBuilder.Append(line);
     }
 
-    private void EmitAppend(Type type)
+    private void EmitAppend(Type type, string suffix = " ")
     {
         if (type.IsConstant)
-            EmitAppend("const ");
+            EmitAppend("const" + suffix);
         
         switch (type.ActualType)
         {
             case TypeVal.Number:
-                EmitAppend("double ");
+                EmitAppend("double" + suffix);
                 break;
             case TypeVal.String:
-                EmitAppend("String ");
+                EmitAppend("String" + suffix);
                 break;
             case TypeVal.Boolean:
-                EmitAppend("boolean ");
+                EmitAppend("boolean" + suffix);
                 break;
             case TypeVal.Blank:
-                EmitAppend("void ");
+                EmitAppend("void" + suffix);
                 break;
         }
     }
+
+    private void EnterScope() {
```

Comment:
Der skal helt sikkert noget dokumentation til det her - think of the children!

Reply:
I am
![EliteSquadRichGIF](https://user-images.githubusercontent.com/74593818/165523503-55f44257-7b7f-495c-a00e-0e7158e3dc91.gif)